### PR TITLE
support parsing `analyze table t`

### PR DESCRIPTION
**Pull request by JCOR11599**
Adding support to parse `analyze` statements: https://dev.mysql.com/doc/refman/8.0/en/analyze-table.html

### CODE REVIEW

File: go/vt/sqlparser/parse_test.go [hunk 1]
```diff
@@ -1360,7 +1360,7 @@ var (
 			output: "alter table a drop index b",
 		}, {
 			input:  "analyze table a",
-			output: "alter table a",
+			output: "analyze table a",
```
[the remark this answers is not in the document]
Comment (JCOR11599) [reply go/vt/sqlparser/parse_test.go:1363]: I definitely could. I just wanted to keep things simple for the much larger changes to GMS and dolt that depend on this syntax.

[the remark this answers is not in the document]
Comment (max-hoffman) [reply go/vt/sqlparser/parse_test.go:1363]: it's one extra rule, should just add it

[the remark this answers is not in the document]
Comment (JCOR11599) [reply go/vt/sqlparser/parse_test.go:1363]: Should I also worry about the `ON COLUMN c` portion of the syntax as well?
MySQL docs for reference: https://dev.mysql.com/doc/refman/8.0/en/analyze-table.html

[the remark this answers is not in the document]
Comment (max-hoffman) [reply go/vt/sqlparser/parse_test.go:1363]: it looks like that's only for the histograms commands? which seem less salient right now

**Review by fulghum — APPROVED** [review]
Looks good! Thanks for adding syntax support for multiple table names.

**Pull request merged**